src/api/spec/helpers/webui/package_helper_spec.rb

```diff
@@ -2,6 +2,14 @@
 require 'rantly/rspec_extensions'
 
 RSpec.describe Webui::PackageHelper, type: :helper do
+  describe '#removable_file?' do
+    it { expect(removable_file?(file_name: 'foo',               package_name: 'bar')).to                  be true }
+    it { expect(removable_file?(file_name: '_service',          package_name: 'bar')).to                  be true }
+    it { expect(removable_file?(file_name: '_service:sub_file', package_name: 'bar')).to                  be false }
```

Comment:
What happens to the `_service:sub_file` if I remove the _service file?

Reply:
ok

Reply:
Yes, they do. Same for the _product:* files